### PR TITLE
MPOR-6: Allow to update port req on submit error

### PR DESCRIPTION
**Pull request by guillegr123**
The change is to keep the requestId after save, before status change. This will allow to know that the port request is already saved in case the status change fails, in order to update it on further attempts instead of creating a new one.

### CODE REVIEW

File: src/apps/common/submodules/portWizard/portWizard.js [hunk 1]
```diff
@@ -1321,35 +1321,60 @@ define(function(require) {
...
+									stopErrorPropagation: true,
+									success: function(requestId) {
+										waterfallCallback(null, requestId);
+									}
+								}));
+							},
+							function(requestId, waterfallCallback) {
+								// Add request ID to args object, to know that the request
+								// has already been saved at this point
+								args = _.merge({
```
Comment (azefiel) [inline src/apps/common/submodules/portWizard/portWizard.js]: You should use `lodash#set` to assign a value to that property as it takes care of creating those nested objects if missing.

File: src/apps/common/submodules/portWizard/portWizard.js [hunk 2]
```diff
@@ -1475,6 +1500,7 @@ define(function(require) {
 							data: self.portWizardNormalizePortRequestData(args.data.request)
 						},
 						success: function(portRequest) {
+							_.set(args, 'data.request.id', portRequest.id);
```
Comment (azefiel) [inline src/apps/common/submodules/portWizard/portWizard.js]: I believe there is a store for this, so maybe you should save the whole payload:
```suggestion
							self.portWizardSet('portRequest', portRequest);
```

Comment (azefiel) [reply src/apps/common/submodules/portWizard/portWizard.js]: Never mind, doesn't seem like the wizard's steps use the store.

Comment (guillegr123) [reply src/apps/common/submodules/portWizard/portWizard.js]: Yeah, it does not seem to use it.

File: src/apps/common/submodules/portWizard/portWizard.js (hunk 1 above)
Comment (guillegr123) [reply src/apps/common/submodules/portWizard/portWizard.js]: I used `lodash#merge` to not to modify the original `args` object, but I think that it does not really matter, as it probably is modified along the way anyways. I'll change it to `lodash#set`.

**Pull request merged**